src/results/existing_chiller.jl

```diff
@@ -29,42 +29,47 @@
 # *********************************************************************************
 """
 `ExistingChiller` results keys:
-- `year_one_to_tes_series_ton` 
-- `year_one_to_load_series_ton`
+- `production_to_tes_series_ton` 
+- `production_to_load_series_ton`
 - `year_one_electric_consumption_series`
-- `year_one_electric_consumption_kwh`
-- `year_one_thermal_production_tonhour`
+- `annual_electric_consumption_kwh`
+- `annual_thermal_production_tonhour`
+
+!!! note "'Series' and 'Annual' energy outputs are average annual"
```

Comment:
Same response to "AbsorptionChiller" note - we are not currently using any sort of degradation/levelization, so annual average = each year exactly. I don't feel strongly weather you leave in the note or not because it's still true.

Reply:
removed for this!